list_constraints.tf

```diff
@@ -82,10 +82,10 @@ resource "google_organization_policy" "org_policy_list_deny_all" {
   Folder policy, deny all (list constraint)
  *****************************************/
 resource "google_folder_organization_policy" "folder_policy_list_deny_all" {
-  count = "${local.folder && local.list_policy && local.enforce == "true" && var.deny_list_length == 0 ? 1 : 0}"
+  count = local.folder && local.list_policy && local.enforce == true && var.deny_list_length == 0 ? 1 : 0
```

Comment:
please remove "== true"

Reply:
local.enforce can be null, true or false. boolean evaluation of local.enforce fails when it's null